packages/client/src/returns/types/getReturnWorkflow.types.ts

```diff
@@ -0,0 +1,30 @@
+import type { Config } from '../..';
+import type { Return } from './return.types';
+
+export enum ReturnWorkflowStatuses {
```

Comment:
Since this enum is not used as a bitfield or flags, the name should be in singular

```suggestion
export enum ReturnWorkflowStatus {
```